public/js/extra.js

```diff
@@ -845,12 +845,14 @@ const linkifyAnchors = (level, containingElement) => {
   for (let i = 0, l = headers.length; i < l; i++) {
     let header = headers[i]
     if (header.getElementsByClassName('anchor').length === 0) {
-      if (typeof header.id === 'undefined' || header.id === '') {
-                // to escape characters not allow in css and humanize
+      if ((typeof header.id === 'undefined' || header.id === '')) {
```

Comment:
Maybe the parenthesis are not necessary here?

Reply:
Oh, it might because the consistence you want to align with below code?

Reply:
Nope, they were actually just left from experiments. Fixed.